components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -10,14 +10,19 @@
 #include <memory>
 #include <string>
 
+#include "bat/ads/issuers_info.h"
+#include "bat/ads/notification_info.h"
 #include "bat/ledger/ledger.h"
 #include "bat/ledger/wallet_info.h"
 #include "base/files/file_path.h"
 #include "base/observer_list.h"
 #include "base/memory/weak_ptr.h"
+#include "bat/confirmations/confirmations_client.h"
 #include "bat/ledger/ledger_client.h"
 #include "brave/components/services/bat_ledger/public/interfaces/bat_ledger.mojom.h"
 #include "brave/components/brave_rewards/browser/rewards_service.h"
+#include "brave/components/brave_ads/browser/ads_service.h"
```

Comment:
there is no reason for these to be here. Only add includes to the header file if they are needed for the declaration. Everything else should go in the cc file